src/main/resources/openapi/annotations.yaml

```diff
@@ -51,6 +51,77 @@ paths:
               schema:
                 $ref: './problem.yaml'
 
+  /annotations/{annotation_id}/documents/{annotation_document_id}:
```

Comment:
When adding a new endpoint, can you add an example request to the postman collection?

Reply:
Sure. Will do.